03-functions-and-loops/invest.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+def invest(initial_amount, interest_rate, duration):
```

Comment:
The name of the function doesn't suggest what it really does. I think something like `print_invest_prediction` would be more appropriate.